extensions/wikidata/module/scripts/dialogs/schema-alignment-dialog.js

```diff
@@ -818,16 +820,22 @@ SchemaAlignmentDialog._initField = function(inputContainer, mode, initialValue,
     });
     if (mode === "amount") {
         input.attr("placeholder", $.i18n._('wikidata-schema')["amount"]);
+        SchemaAlignmentDialog.setupStringInputValidation(input, /^[\-+]?\d+(\.\d*)?(E[\-+]\d+)?$/);
     } else if (mode === "url") {
         input.attr("placeholder", $.i18n._('wikidata-schema')["full-url"]);
+        SchemaAlignmentDialog.setupStringInputValidation(input, /^https?:\/\/.+$/);
```

Comment:
Is this always https ?  or can it fall back to http sometimes ?

Reply:
that is a regular expression which validates URLs - as you can see there is a `?` after the `s`, which means that `s` can be omitted - so yes, both http and https are accepted in this input.